tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -3465,3 +3465,362 @@ def create_zeros_like_3_val():
     assert(     f3_shape() ==      create_zeros_like_3_shape()      )
     assert(isclose(     f3_val()  ,      create_zeros_like_3_val()        , rtol=RTOL, atol=ATOL))
     assert(type(f3_val())  == type(create_zeros_like_3_val().item()))
+
+def test_numpy_real_scalar(language):
+
+    @types('bool')
+    def test_bool(a):
+        import numpy as np
+        b = np.real(a)
+        return b
+
+    @types('int')
+    def test_int(a):
+        import numpy as np
+        b = np.real(a)
+        return b
```

Comment:
What is the return type in these cases? Could you add type checks to ensure that we calculate the result type correctly please

Reply:
sure

Reply:
You re-requested a review, but this hasn't been done, has it?

Reply:
sorry, i forgot it.

Reply:
Please re-read all unresolved conversations before re-requesting reviews to avoid the situation where I need to review and just point to open conversations. Thanks